bucket/nircmd.json

```
@@ -1,7 +1,8 @@
 {
     "version": "2.86",
     "homepage": "https://www.nirsoft.net/utils/nircmd.html",
-    "license": "Unknown",
+    "description": "A small command-line utility that allows you to do some useful tasks without displaying any user interface.",
```

I am trying to come up with some neutral description. Feel free to ignore this suggestion.

```suggestion
    "description": "Execute various system wide tasks.",
```